com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs

```diff
@@ -120,6 +120,11 @@ internal set
             }
         }
 
+        /// <summary>
+        /// Override this property to show a link to documentation on the editor toolbar.
+        /// </summary>
+        public virtual string documentationURL => null;
```

Comment:
We only need this for Editor. 
Plus, do we need it outside? I think Protected is enough for this.
Furthermore, here we store one occurence of the documentation link per volume component object. We really do not need to charge the memory with this as the url should be static per volume component.

Reply:
Ah yes, so the reason this was done initially was to support the case of multiple documentations in the case of Raytracing enabled or disabled. For example, AO currently has a different set URL for raytraced / rasterized modes (and this is the case for any volume component with Raytracing support). 

However after discussion with Seb, we will instead fix this on the documentation side and consolidate everything together.

So it should be safe to remove this override completely, we can stick with the attribute (since there should only ever be one URL per component).

Reply:
So actually, brought back the `documentationURL` to pre-process the attribute like you mentioned (saving the overhead on OnGUI).